content/applications/inventory_and_mrp/quality/quality_management/quality_control_points.rst

```diff
@@ -84,8 +84,10 @@ selected:
 - :guilabel:`Pass - Fail` checks specify a criterion that products must meet for the check to pass.
 - :guilabel:`Measure` checks prompt employees to record a measurement of the product that must be
   within a tolerance of a norm value for the check to pass.
-- :guilabel:`Worksheet` checks provide an interactive worksheet that must be filled out by the
-  employee processing the check.
+- :guilabel:`Worksheet` checks require the employee processing the check to fill out an interactive
```

Comment:
Optional: For lines 47-49, you might consider placing them in an example admonition block. This is, of course, personal preference, but I find that using colorful blocks can make content more engaging—when used in moderation, of course!